src/main/scala/xiangshan/frontend/newRAS.scala

```diff
@@ -733,9 +733,8 @@ class RAS(implicit p: Parameters) extends BasePredictor {
   val update_pc = io.update.bits.pc
 
   // To improve Clock Gating Efficiency
-  update.meta := RegEnable(io.update.bits.meta, io.update.valid && (io.update.bits.is_call || io.update.bits.is_ret))
-
-  val updateMeta = update.meta.asTypeOf(new RASMeta)
+  // update.meta := RegEnable(io.update.bits.meta, io.update.valid && (io.update.bits.is_call || io.update.bits.is_ret))
```

Comment:
Please remove the commented code, and change the comments here to describe why   not CGE opt here.
For example "full core power down sequence need xxx to be xxx"

Reply:
Your insights are very insightful. I will modify the code comments in this part.